plugins/inventory/proxmox.py

```diff
@@ -0,0 +1,350 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Guido Günther <agx@sigxcpu.org>, Daniel Lobato Garcia <dlobatog@redhat.com>
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: proxmox
+    plugin_type: inventory
+    short_description: proxmox inventory source
+    version_added: "1.0.0"
+    author:
+      - Jeffrey van Pelt (@Thulium-Drake) <jeffrey.vanpelt@element-networks.nl>
+    requirements:
+        - requests >= 1.1
+    description:
+        - Get inventory hosts from the proxmox service.
+        - "Uses a configuration file as an inventory source, it must end in ``.proxmox.yml`` or ``.proxmox.yaml`` and has a ``plugin: proxmox`` entry."
+        - Will retrieve the first network interface with an IP for Proxmox nodes
+        - Can retrieve LXC/QEMU configuration as facts
+    extends_documentation_fragment:
+        - inventory_cache
+    options:
+      plugin:
+        description: the name of this plugin, it should alwys be set to 'proxmox' for this plugin to recognize it as it's own.
+        required: True
+        choices: ['proxmox']
+        type: str
+      url:
+        description: url to proxmox
+        default: 'http://localhost:8006'
+        type: str
+      user:
+        description: proxmox authentication user
+        required: True
```

Comment:
```suggestion
        required: yes
```
In the web doc booleans will be automatically converted to `yes`/`no`. Wouldn't it be better to make the module and the doc more consistent by using `yes/no` everywhere (and it's a requirement for EXAMPLES section now)